ts-rs/src/lib.rs

```diff
@@ -70,31 +70,31 @@
 //! - type aliases must not alias generic types (#70)
 //!
 //! ## cargo features
-//! - `serde-compat` (default)  
+//! - `serde-compat` (default)
```

Comment:
Seems like something removed the double whitespace at the end of lines in the docs. i think this will screw up line breaks in rustdoc